src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -403,6 +404,12 @@ impl<'a> Linker for GccLinker<'a> {
         };
     }
 
+    fn strip(&mut self) {
+        if self.sess.opts.cg.strip {
```

Comment:
This condition is not linker-specific, so it should be in `link.rs` rather than here.

Reply:
I am not sure if I understand your concern. Did you mean I need to add a function like https://github.com/contrun/rust/blob/942a16ebcc944057ebe87fb0e4cb9755e85d8822/src/librustc_codegen_ssa/back/link.rs#L1309 or I need to store the option `strip` somewhere in `link.rs`? The option itself is not linker-specific, but different linker interpret the option differently. So I added a `strip` for each linker. Can you give an example on what I should do in `link.rs`?

Reply:
I mean the *condition* `if self.sess.opts.cg.strip` should be in `link.rs`
```
// Pass strip flags down to the linker.
if sess.opts.cg.strip {
    cmd.strip();
}
```

Reply:
That is reasonable. I will update it.